### ISSUE TITLE
Fedora 34 build: `CMake Error at cmake/FindSCIP.cmake:68 (set_property): set_property not given a PROPERTY <name> argument.`

### ISSUE THREAD

**Issue by viniul** [google/or-tools#2801]
**What version of OR-Tools and what language are you using?**
Version: stable
Language: C++

**Which solver are you using (e.g. CP-SAT, Routing Solver, GLOP, BOP, Gurobi)**
SCIP

**What operating system (Linux, Windows, ...) and version?**
Fedora 34, cmake version 3.20.5
**What did you do?**
I tried to build the `stable` branch like so:
```
git clone https://github.com/google/or-tools.git && cd or-tools
mkdir build && cd build && SCIP_ROOT=/usr/local/ cmake ..
```

**What did you expect to see**
A successful build
**What did you see instead?**
The build fails with the following error messages:
<pre>
-- SCIP_ROOT: /usr/local/                                                             
-- TARGET_PROPERTY:                                                                       
CMake Error at cmake/FindSCIP.cmake:68 (set_property):                                    
  <b>set_property not given a PROPERTY <name> argument.                                          </b>
Call Stack (most recent call first):                                                  
  cmake/deps.cmake:53 (find_package)                                                      
  CMakeLists.txt:246 (include)          
</pre>

Note: I posted this issue as a problem about linking the pthreads library before. I edited it now, because I think the issue is in fact a different one - namely the Error above.

**Comment by Mizux** [comment]
I think our custom FindSCIP.cmake should be updated (since SCIP is a cmake based build project and your provide its owns).
Currently we are looking for a static build of scip...

Potential workaround: try to `rm cmake/FindSCIP.cmake`

**Comment by viniul** [comment]
Thanks @Mizux !
Removing the `FindSCIP.cmake` did not work -- However, I solved the issue in the meantime by changing the following line in `FindSCIP.cmake` (line 68):
```
    set_property(TARGET SCIP::SCIP TARGET_PROPERTY IMPORTED_LOCATION
```
to:
```
    set_property(TARGET SCIP::SCIP PROPERTY IMPORTED_LOCATION
```
Now I am getting the error:
```
/usr/bin/ld: /usr/local/lib/liblibscip.a(scip_disp.o): relocation R_X86_64_32 against `.LC0' can not be used when making a shared object; recompile with -fPIC
```
when building with `cmake`.

I used `scipoptsuite-7.0.3`, compiled with normal `make`. To able to compile, I linked `ln -s /usr/local/lib/libscip.a /usr/local/lib/liblibscip.a`.

Any idea on how to still get this working? I am currently investigating recompiling the `scipoptsuite-7.0.3` with the necessary build flags, but was hoping for some other solution maybe?
